apps/dc_tools/assets/bootstrap-odc.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
This script is now in two places...

Maybe it should be graduated to a Python file?

Reply:
We could have different ones. The tests are now in 2 places too. We consolidate / create then dedup.